src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -717,71 +717,13 @@ For example, if the page that was copied from is called `Leads`, then the newly
 }
 ----
 
-[#setting-page-icon]
-== Setting page Icon
+== Configuring Page Icons
 
-Icon can be set to a record list or record page.
+Icon can be set to a list or record page.
```

Comment:
This is a bit malformed; some alternatives would be

* `Page icons can be set to list or record pages`
* `An icon can be set to a list or record page`
* `A page icon can be set to a list or record page`